llama_stack/templates/fireworks/run-with-safety.yaml

```diff
@@ -95,62 +95,20 @@ metadata_store:
   db_path: ${env.SQLITE_STORE_DIR:~/.llama/distributions/fireworks}/registry.db
 models:
 - metadata: {}
-  model_id: meta-llama/Llama-3.1-8B-Instruct
+  model_id: ${env.INFERENCE_MODEL}
```

Comment:
Hmm, this run-with-safety template shouldn't be changed. 


cc @vladimirivic Could you help take a look? I think this is introduced incorrectly in https://github.com/meta-llama/llama-stack/blob/91907b714e825a1bfbca5271e0f403aab5f10752/llama_stack/templates/fireworks/fireworks.py#L123-L140. 

It should follow smt like together's run-with-safety template: https://github.com/meta-llama/llama-stack/blob/91907b714e825a1bfbca5271e0f403aab5f10752/llama_stack/templates/together/together.py#L120C32-L141

Reply:
I rebased and re-generated the files again. Not sure if it incorported the recent fixes correctly

Reply:
I think it pulled changes in https://github.com/meta-llama/llama-stack/pull/766.

Reply:
The fireworks templates have been fixed in #766 . But I think the vllm run.yaml without `tool_runtime` still suggest you might have an older version? Could you help double check?

Reply:
Looks like there are indeed some issues with my local python env. I think I fixed them now. Just pushed an update